Gruntfile.js

```diff
@@ -76,6 +81,12 @@ module.exports = function (grunt) {
                     "sudo rm -f /usr/share/applications/gpii-usb-user-listener.desktop",
                     "sudo rm -f -r /var/lib/gpii"
                 ].join("&&")
+            },
+            runAcceptanceTests: {
+                command: "vagrant ssh -c 'DISPLAY=:0 node /home/vagrant/sync/tests/AcceptanceTests.js'"
```

Comment:
From my perspective, it's cool and weird at the same time that nothing "happens" inside the virtual machine when running the acceptance tests. Since I'm used to see the desktop applying/restoring preferences lively when running the tests and I'm pretty new in Vagrant, I'm not sure what's the right behavior here.

In case we want to "see things happening" inside our VM, we should think about adding a mechanism to set the "DBUS_SESSION_BUS_ADDRESS" environment variable to the active's user one prior to running the tests.
In case we don't want to "see things happening", we should, at least, describe this issue in the documentation, and tell developers to run the tests manually inside a VM's gnome-terminal.